hyperscale/002-db-user-creation.sql

```diff
@@ -0,0 +1,2 @@
+create user citus with encrypted password 'Citus123';
```

Comment:
is it possible to configure the password via a setting somehow?

Reply:
Parameterized password. Now we can set the password in docker run

docker run --name image_name -p 5432:5432 -e POSTGRES_PASSWORD="Citus124" -e CITUS_PASSWORD="1234"  hyperscale_image